tests/component-tests/reorderer/js/ImageReordererTests.js

```diff
@@ -13,8 +13,9 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
 
 /* global jqUnit */
 
+"use strict";
+
 (function ($) {
-    "use strict";
 
```

Comment:
Can this be removed?

Reply:
The closure, you mean? There is an obscure timing-related issue about initialising the variable "k" in the inner scope that I didn't feel the need to look into the causes of, so I left it in.

Reply:
ok